packages/blade/tsconfig-generate-types.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": ["**/**.test.ts", "**/**.test.tsx", "__tests__"],
+  "include": ["src", "src/tokens/theme/theme.d.ts"],
```

Comment:
How would we make sure we're not bundling any unnecessary d.ts files that we added for external npm packages (that don't have types)?

Reply:
Also, how would we make sure we're not bundling any unnecessary d.ts files that we added for external npm packages (that don't have types)?

Reply:
that's a typo, removed it. It's not required. It was left over from my experimentation